code/modules/reagents/chemistry/holder.dm

```diff
@@ -700,7 +700,7 @@
 				matches = FALSE
```

Comment:
This `FALSE` here should be `TRUE` as well, otherwise the `strict = TRUE` usage of this proc will still be broken

Reply:
I'll try setting it to TRUE again and seeing if it still works. Funnily enough, I don't think it worked when I had both set to TRUE. But I'll check.

Reply:
So, it worked still. I have little idea why it didn't work last night but 🙃